11ty/definitions/fr_FR/pluriel.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Pluriel
+slug: pluriel
+speech: nom
+defined: true
+---
+
+désigne plus d'une entité vivant dans un organisme d'accueil.
```

Comment:
Suggestion:
```
désigne plusieurs entités
```

Reply:
Yes, has a nicer ring to it: implemented!